AdminUi/apps/admin_ui/lib/core/widgets/filters/date_filter.dart

```diff
@@ -84,8 +84,8 @@ class _DateFilterState extends State<DateFilter> {
   void _clearDate() {
     setState(() {
       _selectedDate = null;
+      widget.onDateSelected(_selectedDate, _operator);
```

Comment:
don't run unnecessary things in setState.

Reply:
Changes made.